--- ext/ffmpeg/libavcodec/Makefile	2019-03-21 00:45:29.000000000 +0000
+++ ext/ffmpeg/libavcodec/Makefile	2019-03-21 01:22:05.431690213 +0000
@@ -932,7 +932,6 @@
             jpeg2000dwt                                                 \
             mathops                                                    \
             options                                                     \
-            avfft                                                       \
 
 TESTPROGS-$(CONFIG_CABAC)                 += cabac
 TESTPROGS-$(CONFIG_FFT)                   += fft fft-fixed fft-fixed32
--- ext/ffmpeg/tests/fate/libavutil.mak	2019-03-21 23:36:01.422273799 +0000
+++ ext/ffmpeg/tests/fate/libavutil.mak	2019-03-21 23:36:06.355805734 +0000
@@ -87,10 +87,6 @@
 fate-parseutils: libavutil/parseutils-test$(EXESUF)
 fate-parseutils: CMD = run libavutil/parseutils-test
 
-FATE_LIBAVUTIL-$(CONFIG_PIXELUTILS) += fate-pixelutils
-fate-pixelutils: libavutil/pixelutils-test$(EXESUF)
-fate-pixelutils: CMD = run libavutil/pixelutils-test
-
 FATE_LIBAVUTIL += fate-random_seed
 fate-random_seed: libavutil/random_seed-test$(EXESUF)
 fate-random_seed: CMD = run libavutil/random_seed-test
--- ext/ffmpeg/libavutil/Makefile	2019-03-21 23:52:07.530962659 +0000
+++ ext/ffmpeg/libavutil/Makefile	2019-03-21 23:52:12.536458587 +0000
@@ -183,7 +183,6 @@
             pca                                                         \
             parseutils                                                  \
             pixdesc                                                     \
-            pixelutils                                                  \
             random_seed                                                 \
             rational                                                    \
             ripemd                                                      \
--- ext/ffmpeg/tests/fate/api.mak	2019-03-22 00:02:30.727198917 +0000
+++ ext/ffmpeg/tests/fate/api.mak	2019-03-22 00:02:39.670724774 +0000
@@ -14,12 +14,6 @@
 fate-api-h264: $(APITESTSDIR)/api-h264-test$(EXESUF)
 fate-api-h264: CMD = run $(APITESTSDIR)/api-h264-test $(TARGET_SAMPLES)/h264-conformance/SVA_NL2_E.264
 
-FATE_API_LIBAVFORMAT-yes += fate-api-seek
-fate-api-seek: $(APITESTSDIR)/api-seek-test$(EXESUF) fate-lavf
-fate-api-seek: CMD = run $(APITESTSDIR)/api-seek-test $(TARGET_PATH)/tests/data/lavf/lavf.flv 0 720
-fate-api-seek: CMP = null
-fate-api-seek: REF = /dev/null
-
 FATE_API_SAMPLES-$(CONFIG_AVFORMAT) += $(FATE_API_SAMPLES_LIBAVFORMAT-yes)
 
 ifdef SAMPLES
